src/app/games/mancala/kalah/KalahTutorial.ts

```diff
@@ -0,0 +1,85 @@
+import { Tutorial, TutorialStep } from 'src/app/components/wrapper-components/tutorial-game-wrapper/TutorialStep';
+import { MancalaState } from '../commons/MancalaState';
+import { KalahMove } from './KalahMove';
+import { MancalaDistribution } from '../commons/MancalaMove';
+import { MGPValidation } from 'src/app/utils/MGPValidation';
+
+export class KalahTutorial extends Tutorial {
+    public tutorial: TutorialStep[] = [
+        TutorialStep.informational(
+            $localize`Mancala`,
+            $localize`Mancala is the name of a family of board games that date back at least to the third century. Mancalas are games of distribution (sowing) and capture. Their goal is to capture the most seeds. The spaces in Mancalas are called the houses. As you're playing Dark, the 6 houses on the bottom are yours.`,
+            MancalaState.getInitialState(),
+        ),
+        TutorialStep.informational(
+            $localize`Kalah`,
+            $localize`Bonus fact: Kalah has been created in America in 1940 by William Julius Champion Jr.`,
+            MancalaState.getInitialState(),
+        ),
+        TutorialStep.fromMove(
+            $localize`Sowing`,
+            $localize`The main move in mancala games is sowing, let's see how seeds are sown. As you're playing Dark, the 6 houses on the bottom are yours.<br/><br>Click on the righter bottom houses to sow the seeds it contains: they will be sown clockwise, one seed per house.<br/><br/>Click on the righter house!`,
+            MancalaState.getInitialState(),
+            [KalahMove.of(MancalaDistribution.FIVE)],
+            $localize`Look at the 4 houses that follow clockwise the one you picked, they now contain 5 seeds. This is how seeds are sown: one by one from the house next to the one they come from, clockwise.`,
+            $localize`Failed. Choose the righter house on the bottom.`,
+        ),
+        TutorialStep.forClick(
+            $localize`The Kalah (1/2)`,
+            $localize`The houses on the extreme left and right, unaligned to the others, are the kalah, yours on the left, the opponent's on the right. When sowing, before passing from your last house to the first of the opponent, you must drop one seed in your kalah. But you won't have to drop seed in your opponent's kalah. When you make a capture, your seeds are put in your kalah.<br/><br/>You're playing Dark. Make a move that pass through your kalah then feed opponent's houses.`,
```

Comment:
are the Kalah, avec une majuscule ? ou "are called the Kalah".
Puis je splitterais avant "yours": ". Yours is on the left, your opponent's on the right"
When you make a capture, *the captured seeds go* in your Kalah

Reply:
up

Reply:
Je dirais pas de majuscule, c'est un nom commun.

Reply:
BGA dit Kalah majusculé en parlant du store 
BGG dit Kalah majusculé également: https://boardgamegeek.com/boardgame/2448/kalah
J'irais donc sur le vote de majorité avec BG*

Reply:
Kalah is all capslocked !